src/main/java/chatter/Chatter.java

```diff
@@ -0,0 +1,69 @@
+package chatter;
+
+import java.io.IOException;
+
+import chatter.command.Command;
+
+/**
+ * Represents chatter.Chatter the chatbot.
+ *
+ * @author Anthony Tamzil
+ * @version CS2103T Individual Project AY2023/24 Semester 1
+ */
+public class Chatter {
+    private TaskList tasks;
+    private Ui ui;
+    private Storage storage;
+
+    /**
+     * A constructor for the chatter.Chatter class
+     */
+    public Chatter(String filepath) {
+        this.ui = new Ui();
+        this.storage = new Storage(filepath);
+        try {
+            this.tasks = new TaskList(storage.readFile());
+        } catch (IOException e) {
+            ui.showLoadingError();
```

Comment:
Good naming to let reader know the details of the function! At first glance, I can already tell that the function call is to display error message for loading, and not just a generic showError